modules/vaos/app/services/eps/draft_appointment_service/service_error.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Eps
+  class DraftAppointmentService
```

Comment:
This looks weird to me to define the same class as above. Could this class be DraftAppointmentServiceError?